bootstrap_node.py

```
@@ -71,8 +105,95 @@ async def persist():
+                            if isinstance(key, str) and key == "global-chat":
+                                try:
+                                    # 토픽 피어 목록 가져와서 정리
+                                    topic_data = server.storage.data.get(key)
+                                    if topic_data:
+                                        peers = json.loads(topic_data)
+                                        # 여기서 더 정교한 검증 로직 추가 가능
+                                        if peers and isinstance(peers, list):
+                                            # 토픽은 유지하고 타임스탬프만 업데이트
+                                            timestamp_data[key] = current_time
+                                            continue
+                                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)